### PR TITLE
Contributor Docs: update gitpod docs and small fix

### PR DESCRIPTION
**Pull request by sleepypioneer**
- [x] No issue for this change, was noticed while running through the onboarding docs
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

@noatamir @phofl PyLadies Hamburg Sprint

### CODE REVIEW

File: doc/source/development/contributing_gitpod.rst [hunk 1]
```diff
@@ -77,6 +77,7 @@ repository:
 
       $ python -m pytest pandas
 
+   Note that this command takes a while to run, so once you've confirmed it's running you may want to cancel it using ctrl-c.
```
Comment (sleepypioneer) [inline doc/source/development/contributing_gitpod.rst:80]: This felt appropriate to add from my experience but can also remove it :)

File: doc/source/development/contributing_gitpod.rst [hunk 2]
```diff
@@ -138,9 +139,9 @@ Rendering the pandas documentation
 ----------------------------------
 You can find the detailed documentation on how rendering the documentation with
 Sphinx works in the :ref:`contributing.howto-build-docs` section. To build the full
-docs you need to run the following command in the docs directory::
+docs you need to run the following command in the :ref:`/doc` directory::
```
Comment (sleepypioneer) [inline doc/source/development/contributing_gitpod.rst:142]: this is wrong sorry, let me change it :)

File: doc/source/development/contributing_gitpod.rst [hunk 3]
```diff
@@ -138,9 +139,9 @@ Rendering the pandas documentation
 ----------------------------------
 You can find the detailed documentation on how rendering the documentation with
 Sphinx works in the :ref:`contributing.howto-build-docs` section. To build the full
-docs you need to run the following command in the docs directory::
+docs you need to run the following command in the /doc directory::
```
Comment (noatamir) [inline doc/source/development/contributing_gitpod.rst:142]:
```suggestion
docs you need to run the following command in the ``/doc`` directory::
```

**Comment by sleepypioneer** [comment]
> No preview found for PR #54292. Did the docs build complete?

@noatamir I am not sure what this means, locally (in gitpod) I could build and view the docs)

**Comment by noatamir** [comment]
trying the auto-preview again. If it doesn't work a screenshot would be fine for me

**Comment by sleepypioneer** [comment]
Screen shots for changes:

Screenshot 2023-07-30 at 18 05 04

----

Screenshot 2023-07-30 at 18 06 25

**Comment by noatamir** [comment]
Thanks @sleepypioneer!

**Review by noatamir — APPROVED** [review]
Thanks @sleepypioneer 💐

**Pull request merged**